index.html

```diff
@@ -59,6 +59,174 @@ <h2 class="primary-header green">
       </p>
       <p id="collapse-action-button"></p>
     </section>
+    <section class="works" id="recent-works">
+      <h1 class="primary-header">
+        My Recent Works
+        <p id="indicator"></p>
+      </h1>
```

Comment:
- [Optional] Please use only one `<h1>` tag in each html file. Kindly make this another heading tag for example `<h2>`. This is best SEO practice. Kindly check [here](https://yoast.com/one-h1-heading-per-post/) for more information.